packages/store/tests/select.spec.ts

```diff
@@ -68,6 +84,70 @@ describe('Select', () => {
 
   const states = [MySubState, MySubSubState, MyState];
 
+  it('should be throw error when forgot to import the ngxs module', () => {
+    try {
+      class SelectComponent {
+        @Select((state: any) => state) public state: Observable<any>;
+      }
+
+      new SelectComponent().state.subscribe();
+    } catch (e) {
+      expect(e.message).toEqual(
+        CONFIG_MESSAGES[VALIDATION_CODE.SELECT_FACTORY_NOT_CONNECTED]()
+      );
+    }
+  });
+
+  it('should be throw error when component is frozen', () => {
```

Comment:
`should throw an exception when the component class is frozen`

Reply:
done